docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst

```diff
@@ -47,27 +47,30 @@ following counters are available:
   getdata_rq, Counter, Number of getdata requests
   create_rq, Counter, Number of create requests
   create2_rq, Counter, Number of create2 requests
+  createcontainer_rq, Counter, Number of createcontainer requests
```

Comment:
Add missing metric names and make metric orders consistent among `*_rq`, `*_resp`, `*_resp_fast`, `*_resp_slow`